indico/modules/events/abstracts/templates/display/abstracts.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'layout/conference_page_base.html' %}
+{% from 'events/abstracts/management/_abstract_list.html' import render_abstract_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
```

Comment:
Usually extends should be the first thing in a template since if affects everything else (internally). While imports don't matter in that regard it's still cleaner to follow that guideline IMO - that way you always see if it's extending or not by looking at the very first line.